tests/ClientTest.php

```diff
@@ -210,6 +207,24 @@ public function join($timeout = null)
     }
 }
 
+class Dummy_Serializer implements \Raven\SerializerInterface
```

Comment:
I don\'t know how to create method with mock object library. That's why I never used it in my tests